### PR TITLE
Replace koa-graphql with Graphq-helix

### PR DESCRIPTION
**Pull request by LeAlencar**
This PR aims to replaces koa-graphql with graphql-helix. This is a first implementation and I'm looking for review and contributions with it.

### CODE REVIEW

File: apps/server/src/app.ts [hunk 1]
```diff
@@ -97,6 +65,40 @@ router.all(
...
+  const { user } = await getUser(ctx.header.authorization);
+  const request = {
+    body: ctx.request.body,
+    headers: ctx.req.headers,
+    method: ctx.request.method,
+    query: ctx.request.query,
+  };
+
+  if (shouldRenderGraphiQL(request)) {
+    ctx.body = renderGraphiQL({});
```
Comment (sibelius) [inline apps/server/src/app.ts:78]: why {} ?

**Review by noghartt — APPROVED** [review]
cool

File: apps/server/src/app.ts (hunk 1 above)
Comment (LeAlencar) [reply apps/server/src/app.ts:78]: Just following the docs, I don't think it's neccessary.
 https://github.com/contra/graphql-helix/blob/main/examples/koa/server.ts#L26

Comment (sibelius) [reply apps/server/src/app.ts:78]: can we use early return instead?

**Review by sibelius — APPROVED** [review]
tests?

File: apps/server/src/app.ts (hunk 1 above)
Comment (LeAlencar) [reply apps/server/src/app.ts:78]: Yes, I think so.

**Pull request merged**